packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -74,6 +74,7 @@ export const commentCountByDiscussionId = (parent: RootDataLoader) => {
         r
           .table('Comment')
           .getAll(r.args(discussionIds as string[]), {index: 'discussionId'})
+          .filter({isActive: true})
```

Comment:
Also leftover from when I was using commentCount as number of replies instead of the number of threadables.  Doesn't impact any existing features (since the meeting summary uses a comment count attached to the meeting object on meeting completion) but is still a latent bug where deleted comments would still be counted for the `Discussion` object